packages/react/src/theme.js

```diff
@@ -10,10 +10,20 @@ export const animation = {
   long: 'cubic-bezier(.4, 0, .2, 1)'
 }
 
+const createTransition = (properties, s) => {
+  if (Array.isArray(properties)) {
```

Comment:
what's the point of checking if `properties` is an array if always it's passed as an array?

Reply:
Yup! You're right, it was from an earlier version I did where I was passing either an array or a string, until I realized I could just destructure the parameters 😅